app/src/main/java/io/homeassistant/companion/android/webview/WebViewActivity.kt

```diff
@@ -259,12 +268,19 @@ class WebViewActivity : AppCompatActivity(), io.homeassistant.companion.android.
         super.onDestroy()
     }
 
-    override fun showError() {
+    override fun showError(isAuthenticationError: Boolean) {
+        if (isShowingError)
+            return
+        isShowingError = true
```

Comment:
Should this ever be set to `false` ?

Reply:
This is so that when the webview fails to load it doesn't also show an alert.

Reply:
Sorry, I wasn't clear. I meant with my comment if we should set it back to `false` once the error is done showing ?